recipes/tbb/all/conanfile.py

```diff
@@ -41,14 +41,14 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-            
+
     def validate(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self):
-            raise ConanInvalidConfiguration("Cross building is not yet supported. Contributions are welcome")
-        if self.settings.os == "Macos" and \
-           self.settings.compiler == "apple-clang" and \
-           tools.Version(self.settings.compiler.version) < "8.0":
-            raise ConanInvalidConfiguration("%s %s couldn't be built by apple-clang < 8.0" % (self.name, self.version))
+        if self.settings.os == "Macos":
+            if tools.cross_building(self):
```

Comment:
```suggestion
            if hasattr(self, "settings_build") and tools.cross_building(self):
```

Reply:
you can see in the status checks of this PR `[required] tbb/2019_u9@ macOS, Clang — All green! (0)`, your changed disabled all macos builds, even native compilation. It should be fixed by testing self.settings_build